dev-docs/bidders/newdream.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: bidder
+title: Newdream
+description: Prebid Newdream Bidder Adapter
+biddercode: newdream
+aliasCode : appnexus
+media_types: video,banner,native
+tcfeu_supported: true
+dsa_supported: true
+prebid_member: true
+userIds: all (with commercial activation)
+schain_supported: true
+coppa_supported: true
+usp_supported: true
+gpp_supported: true
```

Comment:
```suggestion
```

Please use `gpp_sids` instead as `gpp_supported` is deprecated. Add the legislations in GPP you support